src/include/dos2linux.h

```diff
@@ -170,7 +170,6 @@ extern int sda_cur_drive_off;
 
 #define lol_dpbfarptr(lol)		(rFAR_FARt(READ_DWORD((lol)+lol_dpbfarptr_off)))
 #define lol_cdsfarptr(lol)		(rFAR_FARt(READ_DWORD((lol)+lol_cdsfarptr_off)))
-#define lol_last_drive(lol)		(READ_BYTE((lol)+lol_last_drive_off))
```

Comment:
Why to remove?

Reply:
Because not all DOS have lastdrive value set in LOL, removing it is a way to ensure we use the lastdrive variable instead.